Pokemon Unity/Assets/Scripts2/Pokemon/Pokemon.cs

```diff
@@ -221,19 +221,19 @@ public Pokemon()
             //calcStats();
         }
 
-        /// <summary>
-        /// Uses PokemonData to initialize a Pokemon from base stats
-        /// </summary>
-        /// <param name="pokemon"></param>
-        /// ToDo: Inherit PokemonData 
-        public Pokemon(Pokemons pokemon) : this()
-        {
-            _base = PokemonData.GetPokemon(pokemon);
-            Exp = new Experience(GrowthRate);
-            Ability = abilityFlag;
-            Gender = gender; //GenderRatio.//Pokemon.PokemonData.GetPokemon(pokemon).MaleRatio
-            eggSteps = _base.HatchTime;
-            GenerateMoveset();
+	/// <summary>
```

Comment:
Lol, are the space changes needed? I swear, if it's going from tabs to spaces (spacebar) I'm ignoring xD